setup.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
While bumping the Python version, it could be useful to remove that obsolete `import unicode_literals`.